students/JaNiece/contact.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="index.css">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title> My Webpage </title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <img src="images/sm.jpeg" class="logo">
+      <a href="index.html"> Home </a>
+      <a href="life.html"> Learn More </a>
+      <a href="goals.html"> My Goals </a>
+      <a href="contact.html" class= "right"> Contact </a>
```

Comment:
🙃 Similar here with the `class` attribute... avoid the extra space between the `=` and the `"`